pkg/driver/node.go

```diff
@@ -714,6 +718,10 @@ func (d *nodeService) getVolumesLimit() int64 {
 		return d.driverOptions.volumeAttachLimit
 	}
 
+	if util.IsSBE(d.metadata.GetRegion()) {
+		return 10
```

Comment:
Can you store this 10 as a constant up top? Something like 

```
sbeDeviceVolumeAttachmentLimit = 10
```

Reply:
Ack, made changes.